usr/lib/config/cfglex.l

```diff
@@ -17,8 +17,12 @@
 #include "cfgparse.h"
 #include "configuration.h"
 
-/* Ignore -Wsign-compare for autogenerated code. */
+/* Ignore some warnings for autogenerated code. */
+#if defined(_AIX)
+#pragma info (nocmp,noobs,nogen)
```

Comment:
indention ?

Reply:
I would prefer option 1.

In general if inside the #ifdef there are other preprocessor statements, then these should be indented.
If inside the #ifdef there is code, then the code is usually already indented (because its inside a function block or similar). So then no additional indention is needed.

At the end its about readability, to spot easily which parts of the code are #ifdef'd and which not.

Reply:
I'll update the coding guidelines doc with this information as well.